tensorflow_privacy/privacy/membership_inference_attack/data_structures.py

```diff
@@ -374,6 +374,84 @@ def _append_array_shape(arr: np.array, arr_name: str, result):
     result.append(' %s with shape: %s,' % (arr_name, arr.shape))
 
 
+def _is_generator(gen, gen_name):
```

Comment:
This function checks that gen is iterator, maybe to name _is_iterator
gen-> obj? (it's not neccasary that the argument is gen, so it's better to name more generic)